### PR TITLE
Accessor for `NativeItem` and `NativeMenu` text element. 

### PR DESCRIPTION
**Pull request by CENSOR1337**
Make an accessor for text element in purpose of font changing.

### CODE REVIEW

File: LemonUI.Example/Demo.cs [hunk 1]
```diff
@@ -234,6 +241,32 @@ private static void RemoveRandom_Activated(object sender, EventArgs e)
             submenu.Remove(item => item != addRandom && item != removeRandom);
         }
 
+        private static void RandomFont_Activated(object sender, EventArgs e)
+        {
+            // Here we can change global font;
+            Array values = Enum.GetValues(typeof(Font));
+            Random random = new Random();
+            Font randomFont = (Font)values.GetValue(random.Next(values.Length));
+            int FontID = (int)randomFont;
```
Comment (justalemon) [inline LemonUI.Example/Demo.cs:250]: There is no need to do an extra cast, as you can use randomFont as usual.

```suggestion
```

File: LemonUI/Menus/NativeMenu.cs [hunk 2]
```diff
@@ -442,6 +442,50 @@ public bool Visible
...
         public ScaledText Title { get; set; }
         /// <summary>
+        /// The font of title menu text.
+        /// </summary>
+        public Font TitleMenuFont
+        {
+            get => Title.Font;
+            set
+            {
+                Title.Font = value;
```
Comment (justalemon) [inline LemonUI/Menus/NativeMenu.cs:452]: You could do the same as you did in NativeItem.cs to make it more compact.

File: LemonUI/Menus/NativeSubmenuItem.cs [hunk 3]
```diff
@@ -29,6 +28,16 @@ public NativeSubmenuItem(NativeMenu menu, NativeMenu parent) : base(menu.Subtitl
             Menu.Parent = parent ?? throw new ArgumentNullException(nameof(parent));
             Activated += NativeSubmenuItem_Activated;
         }
+        /// <summary>
```
Comment (justalemon) [inline LemonUI/Menus/NativeSubmenuItem.cs:31]: Isn't this already present as part of the merge of #25?

File: LemonUI.Example/Demo.cs [hunk 4]
```diff
@@ -86,6 +90,7 @@ public static class Basics
         private static readonly NativeCheckboxItem useMouse = new NativeCheckboxItem("Use Mouse", "If the mouse should be used for selecting different items.", menu.UseMouse);
         private static readonly NativeItem flip = new NativeItem("Flip", "Flips the Menu from the left to the Right.");
         private static readonly NativeItem showHack = new NativeItem("Show Hack", "Shows the hacking minigame.");
+        private static readonly NativeItem randomFont = new NativeItem("RandomFont", "Random font to all text.");
```
Comment (justalemon) [inline LemonUI.Example/Demo.cs:93]: Maybe a better Title and Description that explain stuff more clearly?

```suggestion
        private static readonly NativeItem randomFont = new NativeItem("Set a Random Font", "Sets a random font for all of the Elements in the menu.");
```

File: LemonUI.Example/Demo.cs [hunk 5]
```diff
@@ -234,6 +241,32 @@ private static void RemoveRandom_Activated(object sender, EventArgs e)
...
+                x.TitleMenuFont = randomFont;
+                x.SubtitleFont = randomFont;
+                x.DescriptionFont = randomFont;
+                x.ItemCountFont = randomFont;
+
+                // Title and AltTitle of list
+                x.Items.ForEach(y =>
+                {
+                    /// You can also use font id that already been registered
+                    y.TitleFont = (Font)FontID;
```
Comment (justalemon) [inline LemonUI.Example/Demo.cs:263]:
```suggestion
                    y.TitleFont = randomFont;
```

File: LemonUI.Example/Demo.cs [hunk 6]
```diff
@@ -234,6 +241,32 @@ private static void RemoveRandom_Activated(object sender, EventArgs e)
...
+                    y.AltTitleFont = (Font)FontID;
```
Comment (justalemon) [inline LemonUI.Example/Demo.cs:264]:
```suggestion
                    y.AltTitleFont = randomFont;
```

**Review by justalemon — CHANGES_REQUESTED** [review]
There are some things that don't feel right.

File: LemonUI/Menus/NativeSubmenuItem.cs (hunk 3 above)
Comment (CENSOR1337) [reply LemonUI/Menus/NativeSubmenuItem.cs:31]: Yeah this is wierd two I've do file diff but it's turn out they are identical.

**Review by justalemon — APPROVED** [review]
Looks ok. Ill make some changes myself and then merge.

**Pull request closed without merging**